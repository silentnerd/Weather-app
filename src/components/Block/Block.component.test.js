import React from 'react';
import renderer from 'react-test-renderer';
import Block from './';

describe('Block Component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <Block/>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
