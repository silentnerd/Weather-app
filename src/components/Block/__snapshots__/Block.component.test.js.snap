// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Block Component renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "flex": 1,
      },
      undefined,
      false,
      undefined,
      undefined,
      undefined,
      undefined,
      undefined,
      undefined,
      undefined,
      undefined,
      undefined,
      undefined,
      undefined,
      undefined,
      undefined,
      undefined,
      undefined,
      undefined,
      undefined,
    ]
  }
/>
`;
